rayon-core/src/lib.rs

```diff
@@ -63,6 +63,9 @@ mod unwind;
 mod util;
 
 pub use thread_pool::ThreadPool;
+pub use thread_pool::current_num_threads;
```

Comment:
There's already a `current_num_threads` below, and it's a little different because it reports on the global pool if you're not in a pool already.